boggle.js

```diff
@@ -15,17 +20,56 @@ function printBoggleBoard(boggleBoard) {
   }
 }
 
+// function shuffle(allArrays) {
+//   for (let a = 0; a <= 7; a++) {
+//     let randomDie = Math.floor(Math.random() * 16);
+//     allArrays.push(allArrays[randomDie]);
+//     allArrays.splice(randomDie, 1);
+//   }
+//   console.log(allArrays);
+// }
+
+function randomRange(length) {
+  return Math.floor(Math.random() * length - 1);
```

Comment:
Run this 100 times with the same input and check that it returns everything you expect.